### DISCUSSION TITLE
Understanding relationship extraction from spacy projects

### DISCUSSION THREAD

**Discussion by thalishsajeed** [explosion/spaCy#7556] [Help: Coding & Implementations] [answered]
What do [this](https://github.com/explosion/projects/blob/af28fb27fdf992abcb062d74c52be4c8fb6f281d/tutorials/rel_component/scripts/parse_data.py#L12) block of code in `parse_data.py` stand for? 

```
SYMM_LABELS = ["Binds"]
MAP_LABELS = {
    "Pos-Reg": "Regulates",
    "Neg-Reg": "Regulates",
    "Reg": "Regulates",
    "No-rel": "Regulates",
    "Binds": "Binds",
```

Trying to set up a relationship extraction pipeline and my relations are  - `LIVES_IN,PERSON_ENTITY,GEO_ENTITY` and  `WORKS_AT,PERSON_ENTITY,ORG_ENTITY`

**Comment by polm** [comment, marked as answer]
If you look at the data in the repo, you'll see that the keys in `MAP_LABELS` are used in the raw data. In `parse_data.py` those labels are mapped to either "Regulates" or "Binds". You can learn more about the data in the video introduction:

https://www.youtube.com/watch?v=8HL-Ap5_Axo
